src/components/Hexagons/Hexagon.svelte

```diff
@@ -0,0 +1,29 @@
+<div class="clip-ring h-full w-full bg-white">
+  <div class="clip-outer-hexagon h-full w-full">
+    <div class="clip-hexagon h-full w-full">
+      <slot />
+    </div>
+  </div>
+</div>
+
+<style>
+  .clip-ring,
+  .clip-outer-hexagon {
+    aspect-ratio: 316 / 270; /* from Figma */
+    mask-size: cover;
+  }
+
+  .clip-ring {
+    clip-path: polygon(0% 50%, 25% 100%, 75% 100%, 100% 50%, 75% 0%, 25% 0%);
+    mask-image: url('/masks/hexagon/background.svg');
+  }
+
+  .clip-outer-hexagon {
+    padding: 5.8% 6.9%; /* Magic Numbers */
+    mask-image: url('/masks/hexagon/foreground.svg');
+  }
+
+  .clip-hexagon {
+    clip-path: polygon(0% 50%, 25% 100%, 75% 100%, 100% 50%, 75% 0%, 25% 0%);
```

Comment:
Did you consider using css var to avoid repeating the polygon values?

Reply:
I changed the classes in the Hexagon.svelte file to be more tailwind-like!